pkg/tools/otelcol-config/installation_token_test.go

```diff
@@ -72,6 +74,44 @@ func TestSetInstallationTokenAction(t *testing.T) {
 			ExpWrite:       []byte("SUMOLOGIC_INSTALLATION_TOKEN=abcdef\n"),
 			SystemdEnabled: true,
 		},
+		{
+			Name:  "launchd",
+			Flags: []string{"--set-installation-token", "abcdef"},
+			Launchd: fstest.MapFS{
+				launchdConfigPlist: &fstest.MapFile{
+					Data: []byte(defaultLaunchdConfigXML),
+				},
+			},
+			ExpWrite: []byte(xmlPreamble +
+				`<plist version="1.0">` +
```

Comment:
just curious, how was this created? 🤔

Reply:
FYI, when using backticks, there is no need to concatenate multiple lines. Newlines are included in backtick strings.

Reply:
I was trying to avoid the line wrapping that happens when using backticks with multiple lines. Backticks also prevent having to escape double-quotes.

Reply:
> just curious, was this auto generated? 🤔
> 
> ```
> <plist version="1.0">` +
> `<dict>` +
> `<key>EnvironmentVariables</key>` +
> `<dict>` +
> `<key>SUMOLOGIC_INSTALLATION_TOKEN</key>` +
> ```

Yes, this comes from the packaging repo and is installed by the packages.